platform/core/snackbars/fds-snackbars.module.js

```diff
@@ -38,6 +32,10 @@ import {
 } from './services/snackbar.service';
 
 import { FdsCoasterComponent } from './coaster/coaster.component';
+import {MatSnackBarModule} from "@angular/material/snack-bar";
```

Comment:
This is nit picky, but I think the imports statements should be formatted like the other existing statements with a space between the curly brackets and the module name.

Reply:
ty... not sure why the linter didn't complain...

Reply:
ok I updated the linter and addressed the lint errors it found. plz confirm